website/docs/d/oracledatabase_exadata_infrastructure.html.markdown

```diff
@@ -0,0 +1,146 @@
+
+* `compute_count` - The number of compute servers for the cloud Exadata infrastructure.
+
+* `cpu_count` - The total number of CPU cores allocated.
+
+* `customer_contacts` - A `customer_contacts` block as defined below.
+
+* `data_storage_size_in_tbs` - Size, in terabytes, of the DATA disk group.
+
+* `db_node_storage_size_in_gbs` - The local node storage allocated in GBs.
+
+* `db_server_version` - The software version of the database servers (dom0) in the cloud Exadata infrastructure. Example: `20.1.15`
+
+* `display_name` - The user-friendly name for the cloud Exadata infrastructure resource. The name does not need to be unique.
+
+* `estimated_patching_time` - A `estimated_patching_time` block as defined below.
+
+* `last_maintenance_run_id` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the last maintenance run.
+
+* `lifecycle_details` - Additional information about the current lifecycle state.
+
+* `lifecycle_state` - CloudExadataInfrastructure lifecycle state.
+
+* `location` - The Azure Region where the Exadata Infrastructure exists.
+
+* `maintenance_window` - A `maintenance_window` block as defined below.
+
+* `max_cpu_count` -  The total number of CPU cores available.
+
+* `max_data_storage_in_tbs` - The total available DATA disk group size.
+
+* `max_db_node_storage_size_in_gbs` - The total local node storage available in GBs.
+
+* `max_memory_in_gbs` - The total memory available in GBs.
+
+* `memory_size_in_gbs` - The memory allocated in GBs.
+
+* `monthly_db_server_version` - The monthly software version of the database servers (dom0) in the cloud Exadata infrastructure. Example: `20.1.15`
+
+* `monthly_storage_server_version` - The monthly software version of the storage servers (cells) in the cloud Exadata infrastructure. Example: `20.1.15`
+
+* `next_maintenance_run_id` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the next maintenance run.
+
+* `oci_url` - The URL of the resource in the OCI console.
+
+* `ocid` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the Exadata infrastructure.
+
+* `provisioning_state` - CloudExadataInfrastructure provisioning state
+
+* `shape` - The model name of the cloud Exadata infrastructure resource.
+
+* `storage_count` - The number of storage servers for the cloud Exadata infrastructure.
+
+* `storage_server_version` - The software version of the storage servers (cells) in the Exadata infrastructure. Example: `20.1.15`
+
+* `system_data` - A `system_data` block as defined below.
+
+* `tags` - A mapping of tags assigned to the Exadata Infrastructure.
+
+* `time_created` - The date and time the cloud Exadata infrastructure resource was created.
+
+* `total_storage_size_in_gbs` -  The total storage allocated to the cloud Exadata infrastructure resource, in gigabytes (GB).
+
+* `zones` - The Exadata infrastructure Azure zones.
```

Comment:
```suggestion
* `ocid` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the Exadata infrastructure.

* `provisioning_state` - Cloud Exadata Infrastructure provisioning state

* `shape` - The model name of the Cloud Exadata Infrastructure resource.

* `storage_count` - The number of storage servers for the Cloud Exadata Infrastructure.

* `storage_server_version` - The software version of the storage servers (cells) in the Cloud Exadata Infrastructure. Example: `20.1.15`

* `system_data` - A `system_data` block as defined below.

* `tags` - A mapping of tags assigned to the Cloud Exadata Infrastructure.

* `time_created` - The date and time the Cloud Exadata Infrastructure resource was created.

* `total_storage_size_in_gbs` -  The total storage allocated to the Cloud Exadata Infrastructure resource, in gigabytes (GB).

* `zones` - The Cloud Exadata Infrastructure Azure zones.
```

Reply:
Changed to Cloud Exadata Infrastructure.